### ISSUE TITLE
Training epchs

### ISSUE THREAD

**Issue by algoterranean** [lukasruff/Deep-SVDD#12]
Hello. I have a question regarding the number of epochs you are training for. In the paper, you mention that DCAE is trained for "250+100 epochs" and DSVDD is trained for "150+100" epochs.

Does this mean that:

- For DCAE, you trained for 250 total epochs, with a learning rate adjustment at epoch 150?
- For DSVDD, you pretrained the AE for 150 total epochs (pretraining), with a LR adjustment at epoch 50, then trained the DSVDD portion for 150 additional epochs, with a LR adjustment at epoch 50?
